GPU/GPUTracking/utils/qconfig_helpers.h

```diff
@@ -42,6 +42,11 @@ inline std::string print_type<char>(char val)
   return std::to_string(val);
 };
 template <>
+inline std::string print_type<signed char>(char val)
```

Comment:
should be `signed char val` to be consistent :)

Reply:
```suggestion
inline std::string print_type<signed char>(signed char val)
```